src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -17,6 +17,7 @@
 public class AddCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "add";
+    public static final String COMMAND_ALIAS = "a";
```

Comment:
Nice work with the command alias but remember to also update the user guide to match new behavior.